tools/offline_ota_image_builder/README.md

```diff
@@ -0,0 +1,175 @@
+# Offline OTA image builder
+
+`tools.offline_ota_image_builder` is a helper package for bundling offline OTA image from the OTA images build by ota-metadata(or images compatible with OTA image specification).
+Built image can be used as external cache source for otaproxy, or used by full offline OTA helper util to trigger fully offline OTA update.
+
+**NOTE**: full offline OTA helper util is not yet implemented.
+
+It provides the following features:
+
+1. build offline OTA image from several OTA images,
```

Comment:
I'd like to fix `several OTA images` like this:
```suggestion
1. build offline OTA image from the OTA images build by ota-metadata(or images compatible with OTA image specification)
```

Reply:
Updated in [9e61c5e](https://github.com/tier4/ota-client/pull/252/commits/9e61c5e069f7192b4e800e9f8b23b31253414502).